languages/ruby/lib/oso/polar/polar.rb

```diff
@@ -15,6 +33,14 @@ def initialize
         @ffi_polar = FFI::Polar.create
         @host = Host.new(ffi_polar)
         @load_queue = Set.new
+
+        # Register built-in classes.
+        register_class(Boolean, name: 'Boolean')
```

Comment:
```suggestion
        register_class(PolarBoolean, name: 'Boolean')
```

Reply:
To avoid clashes in the global namespace.